.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Notion ticket link
+<!-- Please replace with your ticket's URL -->
+[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)
+
+
+<!-- Give a quick summary of the implementation details, provide design justifications if necessary -->
+## Implementation description
+* 
+
+
+<!-- What should the reviewer do to verify your changes? Describe expected results and include screenshots when appropriate -->
+## Steps to test
+1.
```

Comment:
Could you please replace this section with a "Notes" section? Just generic bullet list of things that reviewers should keep in mind, things that will be fixed in the future, things that were left out, etc

Reply:
Addressed!